README.md

```diff
@@ -90,7 +90,7 @@ import '@webex/components/dist/webexComponents.css';
 
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {WebexJSONAdapter, WebexAvatar} from '@webex/components';
+import {WebexJSONAdapter, WebexAvatar, WebexDataProvider} from '@webex/components';
```

Comment:
could it be 
```js
import {WebexAvatar, WebexDataProvider, WebexJSONAdapter} from '@webex/components';
```

not a blocker

Reply:
There wasn't an order previously, otherwise I would have. Fixing.